packages/web/src/components/spa/HackAUBG/MentorsSection/MentorsSection.jsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import axios from 'axios';
+import { useState } from 'react';
+import { useEffect } from 'react';
+import { url } from '../../../../Global';
+import { MentorsCard } from './MentorsCard';
+import './mentors_section.css';
+
+export const MentorsSection = () => {
+    const [mentor, setMentors] = useState([{}]);
+
+    const getMentors = () => {
+        axios({
+            method: 'get',
+            url: url + '/api/mentors'
+        })
+            .then((res) => {
+                setMentors(res.data.data.data);
+            })
+            .catch((err) => {
+                console.log(err);
+            });
+    };
+
+    const handleClick = (event) => {
+        return () => {
+            window.open(event, '_blank', 'noopener noreferrer');
+        };
+    };
+
+    const renderMap = () => {
```

Comment:
I know this has been taken from the admin page but it's a terrible component name whatsoever. Please rename it to something better